FWCore/Integration/plugins/AcquireIntESProducer.cc

```diff
@@ -31,6 +31,10 @@
 #include <unistd.h>
 #include <vector>
 
+constexpr int kAcquireTestValue = 11;
+constexpr int kAcquireTestValueUniquePtr1 = 101;
+constexpr int kAcquireTestValueUniquePtr2 = 102;
```

Comment:
Let's put these in anonymous namespace to avoid even slightest possibility for ODR clashes
```suggestion
namespace {
  constexpr int kAcquireTestValue = 11;
  constexpr int kAcquireTestValueUniquePtr1 = 101;
  constexpr int kAcquireTestValueUniquePtr2 = 102;
}
```

Reply:
Done.